SimTransport/PPSProtonTransport/src/BaseProtonTransport.cc

```diff
@@ -3,20 +3,20 @@
 #include "Utilities/PPS/interface/PPSUnitConversion.h"
 #include <CLHEP/Random/RandGauss.h>
 #include <CLHEP/Units/GlobalSystemOfUnits.h>
-#include <cctype>
 
 BaseProtonTransport::BaseProtonTransport(const edm::ParameterSet& iConfig)
     : verbosity_(iConfig.getParameter<bool>("Verbosity")),
-      bApplyZShift(iConfig.getParameter<bool>("ApplyZShift")),
+      bApplyZShift_(iConfig.getParameter<bool>("ApplyZShift")),
       useBeamPositionFromLHCInfo_(iConfig.getParameter<bool>("useBeamPositionFromLHCInfo")),
       produceHitsRelativeToBeam_(iConfig.getParameter<bool>("produceHitsRelativeToBeam")),
-      fPPSRegionStart_45(iConfig.getParameter<double>("PPSRegionStart_45")),
-      fPPSRegionStart_56(iConfig.getParameter<double>("PPSRegionStart_56")),
+      fPPSRegionStart_45_(iConfig.getParameter<double>("PPSRegionStart_45")),
+      fPPSRegionStart_56_(iConfig.getParameter<double>("PPSRegionStart_56")),
       beamEnergy_(iConfig.getParameter<double>("BeamEnergy")),
       etaCut_(iConfig.getParameter<double>("EtaCut")),
       momentumCut_(iConfig.getParameter<double>("MomentumCut")) {
   setBeamEnergy(beamEnergy_);
```

Comment:
this function is uselessly overwriting `beamEnergy_` with itself.
It can be more conveniently replaced by this simple line here:
```C++
  beamMomentum_ = sqrt(beamEnergy_ * beamEnergy_ - ProtonMassSQ);
```

Reply:
I agree, I forget to change this place but it is very little overhead.